docs/extensions-plugins/jetbrains.mdx

```diff
@@ -155,14 +155,14 @@ To open and edit a previously saved snippet:
 You can adjust your snippet's name, description, and classification from the right-click context menu.
 
 ## Use Suggested Save in JetBrains
-Pieces can intelligently identify and save snippets based on re-use patterns in your JetBrains workflow. Whenever Pieces saves a snippet, you'll see a "Saved a Piece!" confirmation in the bottom right corner of your screen. Your snippet is now saved in Pieces with a default name and automatically classified by language.
+Pieces can intelligently identify and save snippets based on re-use patterns in your JetBrains workflow. Whenever Pieces saves a snippet, you'll see a "Saved a Piece!" confirmation in the bottom right corner of your screen. Your snippet is now saved in Pieces with auto-generated metadata. 
 
 <Image zoom src = "https://storage.googleapis.com/pieces_static_resources/jetbrains_marketplace/gifs/SUGGESTED_SNIPPETS_JETBRAINS.gif"/>
 
-<Info>To opt-out of Suggested Save, go to ```Preferences > Tools > Pieces``` and click on Suggested Save.</Info>
```

Comment:
this is a coding thing and actually should probably be how it is laid out here - no worries ill correct this on another PR